sources/packages/web/src/views/institution/InstitutionProfile.vue

```diff
@@ -1,7 +1,11 @@
 <template>
   <full-page-container>
     <template #header>
-      <header-navigator title="Manage institution" subTitle="Manage Profile" />
+      <header-navigator
+        title="Manage institution"
+        data-cy="manageProfileLandingPage"
```

Comment:
I saw sometimes this type of `data-cy` added to the `header-navigator` and sometimes to the `full-page-container`. What is the idea here?

Reply:
The id's in the header navigator are specifically for the landing page - just to make sure that the user is now navigated to the correct page. The id's in the full page containers are to retrieve the elements where the individual cy-ids cannot be assigned for the dynamic elements

Reply:
I did not follow this part "cannot be assigned for the dynamic elements". Is it about the form.io forms?
For the header, I would still suggest some sort of naming pattern like `manageProfileHeader`.

Reply:
same qn

Reply:
updated it